pkgs/games/koules/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, lib, fetchurl, fetchzip, makeDesktopItem, imake, gccmakedep, libX11, libXext, installShellFiles }:
+
+let
+  desktopItem = makeDesktopItem {
+    desktopName = "Koules";
+    name = "koules";
+    exec = "xkoules";
+    icon = "koules";
+    comment = "Push your enemies away, but stay away from obstacles";
+    categories = [ "Game" "ArcadeGame" ];
+  };
+  debian = fetchzip {
```

Comment:
An extra space, please.

And a better name for this, like `debian-extras`.

Reply:
:+1: will do